quix-frontend/client/src/lib/sql-autocomplete/sql-context-evaluator/types.ts

```diff
@@ -13,14 +13,15 @@ export interface TableInfo {
   type: TableType;
   name: string;
   alias: string;
-  columns: string[];
+  columns: any[];
```

Comment:
dont use any
specify type that can be except string and create type Column = string | Type